Core/GDCore/IDE/Project/ArbitraryResourceWorker.cpp

```diff
@@ -78,19 +80,12 @@ void ArbitraryResourceWorker::ExposeResource(gd::Resource& resource) {
 
 ArbitraryResourceWorker::~ArbitraryResourceWorker() {}
 
-void LaunchResourceWorkerOnEvents(const gd::Project& project,
+void LaunchResourceWorkerOnEvents(gd::Project& project,

```

Comment:
@arthuro555 Found the problem, you're declaring a function, but this is not the same as the one in the header, because you removed the `const`.
Update the header to fix the issue.

Right now you have a function declared in the header, but which is never implemented, and a function implemented here, but which is "just" a free function declared and used nowhere.

Reply:
Btw, do you really need to remove the const here?

Reply:
The GDevelop.js Emscripten build is also complaining about a missing function:
![image](https://user-images.githubusercontent.com/1280130/116442526-7f329600-a84a-11eb-96af-2fde7c9b7a3a.png)

which can give another hint that you've not implemented the function you declared :)